7_react/react1/week2/hw-react-week2/src/components/FormTodo.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+const FormTodo = (props) => {
+  const { handleAddItem } = props;
+  const [description, setDescription] = useState("");
+
+  const handleSubmit = (element) => {
+    element.preventDefault();
+
+    handleAddItem({
+      done: false,
+      id: (+new Date()).toString(),
+      description,
+    });
+
+    setDescription("");
+  };
```

Comment:
Overall, I really like how you implemented this component. You generated a unique id by using the datetime, nice aproach. In the industry the backend would typically assign an id either as a number: `id: 123` or as a [unique id](https://www.uuidgenerator.net/dev-corner/javascript): `id: 'c32d8b45-92fe-44f6-8b61-42c2107dfe87'`

UUIDs are universal unique ids with an enormous number of unique combinations. it means that when you generate a random id it is practically impossible that your id has or will ever be generated before by any system ever made. 

In the words of Wikipedia:

> only after generating 1 billion UUIDs every second for the next 100 years, the probability of creating just one duplicate would be about 50%. The probability of one duplicate would be about 50% if every person on earth owns 600 million UUIDs

Reply:
Awesome, link -> fav